tests/contrib/botocore/test.py

```diff
@@ -67,6 +68,7 @@ class BotocoreTest(TracerTestCase):
 
     def setUp(self):
         patch()
+        patch_submodules(True)
```

Comment:
Does the user have to specify `patch_submodules(True)` explicitly to make sure every submodule is patched?